packages/libs/deer/Makefile.toml

```diff
@@ -0,0 +1,41 @@
+extend = { path = "../../../.github/scripts/rust/Makefile.toml" }
+
+[env]
+CARGO_CLIPPY_HACK_FLAGS = "--workspace --feature-powerset --exclude-features default"
+CARGO_TEST_HACK_FLAGS = "--workspace --feature-powerset --exclude-features default"
+CARGO_RUSTDOC_HACK_FLAGS = ""
+CARGO_DOC_HACK_FLAGS = ""
+
+# deer does not have any unsafe code, therefore we do not need to test for miri
+[tasks.miri]
+disabled = true
+
+[tasks.test]
+dependencies = ['install-rust-src']
+
+# required for deer-macro UI tests
+[tasks.install-rust-src]
+private = true
+install_crate = { rustup_component_name = "rust-src" }
```

Comment:
We don't have UI tests (yet?).

Reply:
That is correct, tho I expect pretty heavy use of UI tests once I start development on the derive macro

Reply:
Let's keep it then 🙂